src/theme/components.ts

```diff
@@ -31,5 +31,23 @@ export const components: ISectionTheme = {
         border: 'transparent',
       },
     },
+
+    codeEditor: {
+      bg: 'white',
+      fg: 'grey',
+      border: 'grey',
+
+      syntax: {
```

Comment:
We might want to add more varieties of colors here. I started off with variables that were used by our dependency `react-simple-code-editor`. They should be sufficient for JS/CSS, but other languages might need some special treatment.